cli/internal/helm/client_test.go

```diff
@@ -0,0 +1,21 @@
+package helm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tj/assert"
```

Comment:
```suggestion
	"github.com/stretchr/testify/require"
	"github.com/stretchr/testify/assert"
```
I really wish there was some way to black list certain repositories...
For some reason VS Code likes to add tj/assert instead of stretchr/testitfy/assert....